framework/src/problems/FEProblemBase.C

```diff
@@ -4245,6 +4245,77 @@ FEProblemBase::execMultiAppTransfers(ExecFlagType type, Transfer::DIRECTION dire
 
     _console << COLOR_CYAN << "\nStarting Transfers on " << Moose::stringify(type)
              << string_direction << "MultiApps" << COLOR_DEFAULT << std::endl;
+
+    // output transfer information
```

Comment:
If we really want detailed information, we should let transfers print themselves